core/templates/pages/exploration-player-page/exploration-player-page.module.ts

```diff
@@ -39,7 +39,8 @@ import { FatigueDetectionService } from './services/fatigue-detection.service';
 
 import 'third-party-imports/guppy.import';
 import 'third-party-imports/midi-js.import';
-import 'third-party-imports/skulpt.import';
+import 'skulpt-dist/skulpt.min.js';
+import 'skulpt-dist/skulpt-stdlib.js';
```

Comment:
Do we need both the imports for this to function? If yes, then please move these into the file: core/templates/third-party-imports/skulpt.import.ts which was deleted in this PR. The reason for doing so is to make sure that when we create a new page and we want to use skulpt-dist, we don't worry about what needs to imported and we can just import the file.

Reply:
@srijanreddy98 thanks for suggesting. updated PTAL!